src/java.base/share/classes/java/lang/Exception.java

```diff
@@ -56,6 +57,7 @@
      * call to {@link #initCause}.
      */
     @SideEffectFree
+    @Unique
```

Comment:
Please write as `public @Unique Exception` to put the return type annotation before the type.

Reply:
I have made the changes in the latest commit